src/app/core/entity-components/entity-subrecord/entity-subrecord/entity-subrecord.component.html

```diff
@@ -97,12 +100,18 @@
               class="table-action-icon"
               aria-label="cancel"
               fontIcon="fa-times-circle"
+              angulartics2On="click"
```

Comment:
Is there a reason why in this case the angulartics are attached to the icon and not to the button like in the other two cases?

Reply:
good find, this was just messily done